store/errors.go

```diff
@@ -25,16 +25,20 @@ import (
 	"net/url"
 )
 
+const (
+	// TwoFactorRequiredError is the error code to be returned if 2FA is needed
+	TwoFactorRequiredError = "two-factor-required"
```

Comment:
Can we please name this as:
* TwoFactorErrKind

The suffix `*Error` is generally used for error types. Not a big deal, but easy to avoid in this case.